### ISSUE TITLE
Next.js import issue

### ISSUE THREAD

**Issue by jamesgasek** [wearemothership/react-fast-scroll-pdf#33]
Hello,

This package has been very useful for our project for rendering pdf on resource-strained devices. Our team is moving to Next from a vanilla react SPA, and we are having the following error when importing and using the package. 

<img width="417" alt="image" src="https://github.com/user-attachments/assets/6709606c-92b5-4ae2-81d3-bda80b49f537">

Is this package intended to work with next.js?

**Comment by dandubya** [comment]
I'm getting the same error after upgrading to 1.3.0. If I downgrade to 1.2.1, I do not get this error.

To fix is, I've had to import as follows:
`import { usePDF } from "../../node_modules/react-fast-scroll-pdf";`

But I'm reluctant to try deploying it this way. All of my other packages import just fine without having to add the relative path.

I have removed the node_modules and build directories, and `npm install`ed the project, and I get the same results.
